charts/opentelemetry-collector/values.yaml

```diff
@@ -110,6 +110,36 @@ presets:
   # Best used with mode = daemonset.
   kubeletMetrics:
     enabled: false
+  # Configures the collector to collect MySQL metrics .
+  mysql:
```

Comment:
I'd suggest having an enabled flag on the MySQL level like `mysql.enabled` where by default it collects metrics and logs.

The default state should be collect logs and metrics, if the user don't want to collect one of those then he needs to disable it.

WDYT?

Reply:
I'm a bit torn on this, because if this was "set and forget" and then it would make sense to enable everything with one parameter. 

But because users need to provide different parameters specifically for metrics and logs (plus, logs collection is taken care of by the generic log file receiver, tin this case it's just extra logs), it already requires extra setup from user anyway. Then I think it's better if the user can also explicitly opt in for each signal. 

If we also had `enabled` parameter on both top level and then for each signal, IMHO there would be more space for confusion and misconfiguration (e.g. What if top level is enabled, but then one signal is disabled? Which takes precedence? etc.)

Reply:
I see your point!
I'm fine using the current strategy 😄